graphql/concrete_notification/fields.ts

```diff
@@ -99,3 +101,53 @@ export class ExtendedFieldsConcreteNotificationResolver {
         return Object.values(byCampaign);
     }
 }
+
+@ObjectType()
+class ConcreteNotificationFormatted {
+    @Field((type) => ID)
+    id: number;
+    @Field()
+    headline: string;
+    @Field()
+    body: string;
+    @Field()
+    notificationClass: string;
+    @Field() // is there a better solution than "string" as type?
```

Comment:
Date - which is transported as an ISO string and then automatically serialized / deserialized by Apollo Client / Prisma